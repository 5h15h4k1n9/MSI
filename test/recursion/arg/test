ack(0, 0)
ack(3, 4)
exit
