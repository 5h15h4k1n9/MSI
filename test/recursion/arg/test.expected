(int) 1
(int) 125
Bye-bye!
